extensions/ql-vscode/src/data-extensions-editor/queries/query.ts

```diff
@@ -0,0 +1,6 @@
+export type Query = {
+  mainQuery: string;
+  dependencies?: {
```

Comment:
I wonder if we should just inline `ExternalApi.qll`, so that we just have a single query file that we run instead of having to store multiple files?

Reply:
I dont think it would be too hard to do that split up later, but I dont feel strongly. It also doesnt seem super complicated to have extra dependency files because we also need to have the synthetic pack. So happy to leave it like this.